snap/snapcraft.yaml

```diff
@@ -208,6 +206,9 @@ parts:
       # Build the redis PHP module
       - source: https://github.com/phpredis/phpredis/archive/3.1.4.tar.gz
         source-checksum: sha256/656cab2eb93bd30f30701c1280707c60e5736c5420212d5d547ebe0d3f4baf71
+      # Build the mcrypt PHP module  
+      - source: https://pecl.php.net/get/mcrypt-1.0.1.tgz
```

Comment:
I think that with the custom php plugin that @kyrofa did, you need to use the repository, not the pecl module. Something like:

`- source: https://github.com/php/pecl-encryption-mcrypt.git`

Should be enough in this case.